bower.json

```diff
@@ -27,18 +27,13 @@
   ],
   "dependencies": {
     "polymer": "^1.6.0",
-    "px-theme": "^3.0.0",
-    "px-buttons-design": "^2.0.0",
     "px-deck-selector": "^2.0.0",
     "px-headings-design": "^1.0.0"
```

Comment:
We're using the `regular` and `zeta` classes for the title and subtitle in the non-deck selector view:

```
      <div id="center" class="headerText">
        <template is="dom-if" if="{{useDeckSelector}}">
          <content select="px-deck-selector"></content>
        </template>
        <template is="dom-if" if="{{!useDeckSelector}}">
          <div id="title" class="regular title truncate">[[title]]</div>
          <template is="dom-if" if="{{subtitle}}">
            <div id="subtitle" class="zeta subtitle truncate">[[subtitle]]</div>
          </template>
        </template>
      </div>
```